Documentation/gettingstarted/hubble-install.rst

```diff
@@ -19,10 +21,12 @@
       .. code-block:: shell-session
 
          export HUBBLE_VERSION=$(curl -s https://raw.githubusercontent.com/cilium/hubble/master/stable.txt)
-         curl -L --fail --remote-name-all https://github.com/cilium/hubble/releases/download/$HUBBLE_VERSION/hubble-darwin-amd64.tar.gz{,.sha256sum}
-         shasum -a 256 -c hubble-darwin-amd64.tar.gz.sha256sum
-         sudo tar xzvfC hubble-darwin-amd64.tar.gz /usr/local/bin
-         rm hubble-darwin-amd64.tar.gz{,.sha256sum}
+         HUBBLE_ARCH=amd64
+         if [ "${HUBBLE_ARCH}" = "amd64" ]; then HUBBLE_ARCH=arm64; fi
```

Comment:
`if [ "${HUBBLE_ARCH}"` should be `if [ "$(uname -m)"`?

And `amd64` should be `arm64`? Or did I miss something?